src/components/Logo.cy.js

```diff
@@ -20,5 +20,6 @@ describe("Logo", () => {
     cy.get("a")
       .should("exist")
       .should("have.attr", "href", "https://greenstand.org/")
+      cy.viewport(800, 500)
```

Comment:
I don't think adjusting the viewport after making assertions buys us anything here.  Can we remove this line?